CHANGES.rst

```diff
@@ -226,6 +226,8 @@ Bug Fixes
   - Fix use of the ``relax`` keyword in ``to_header`` when used to change the
     output precision. [#5164]
 
+  - Fix rotateCD(). [#5189]
```

Comment:
No objections from me but it should probably be deprecated for one version before being removed.